### ISSUE TITLE
When application opens from notification throws Error: There is not route defined...

### ISSUE THREAD

**Issue by quoc-axiadids** [aksonov/react-native-router-flux#2697]
Hello, I am encountering an issue when I have my scenes in a Drawer while receiving a push notification. Once I open the application from the background via the notification then I get the "no route defined for key" error. I tried removing the Drawer, and the app opened fine. Could this be an issue with nesting the scenes?
    ...
    `<Router>`
        `<Scene key="root">`
            `<Drawer hideNavBar contentCompoent={ SideBar }>`
                `<Scene key="HOME" component={ Home } initial/>`
                `<Scene key="CONFIG" component={ Config } />`
            `</Drawer>`
        `</Scene>`
    `</Router>`
    ...

### Version
- react-native-router-flux v4

### Expected behaviour
- App opens when clicking on push notification

### Actual behaviour
- App throws an error saying "There is no route defined for key..."

### Steps to reproduce
For non-obvious bugs, please fork this component, modify Example project to reproduce your issue and include link here.
1. Create an application that gets push notification.
2. Put your scenes inside a drawer.
3. Send a push notification, and open app from notification

** Note: I have only tested on Android.

**Comment by Blapi** [comment]
I see a typo in your code, you wrote `contentCompoent` instead of `contentComponent` ?

When you're not using your app like with a push notification, is Actions.HOME() or Actions.CONFIG() working properly?

I've never used a Drawer in my Android app but if you still have this issue, I'm gonna give it a try

**Comment by quoc-axiadids** [comment]
I apologize, that is a typo in my own writing. The actual code is spelled correctly. 

Yes, I can navigate properly. Everything worked fine until I integrated Push Notifications and tested opening the application in the background.

**Comment by Blapi** [comment]
What is your code when you're handling the Push Notification?

**Comment by quoc-axiadids** [comment]
I override the onMessageReceived function in Java and just print the message.

`@Override`
`public void onMessageReceived(RemoteMessage remoteMessage) {`
`System.out.println("Test" + remoteMessage.getNotification());`
`}`

These are the results in the following states:
foreground - Message is received and seen in logcat
background w/o Drawer - Application is opened successfully and message is not received through the `onMessageReceived` method.
background w/ Drawer - Application is opened and throws "... no route defined for key0 Must be one of key3..."

*Note every additional time I try to open the application via push notification key0 is increment (key1, key2...)

**Comment by quoc-axiadids** [comment]
Another experiment I did was creating a custom drawer component using react-native-drawer.
I wrapped the `<Router>` in a custom-drawer component, and it works fine.

It could be an implementation problem that I have, because I noticed that once I bring the app to the background from anything other than the initial `Scene`. I will also get the same "... no route defined for key0" message.

I would prefer not to expand my dependencies if react-native-router-flux already has logic to support a drawer.
